cypress/integration/RichTextEditor.spec.ts

```diff
@@ -7,9 +7,9 @@ import {
 } from '../../packages/rich-text/src/helpers/nodeFactory';
 
 function expectRichTextFieldValue(expectedValue, editorEvents?) {
-  cy.getRichTextField().then((field) => {
-    expect(field.getValue()).to.deep.eq(expectedValue);
-  });
+  cy.getRichTextField()
+    .then((field) => field.getValue())
+    .should('deep.equal', expectedValue);
```

Comment:
Replaced `expect` with Cypress's `should` to be retried. Which removes the need for random `cy.wait` calls all over the place.